src/main/java/com/ericsson/eiffel/semantics/events/Event.java

```diff
@@ -53,4 +54,22 @@ public Meta getMeta() {
 	public void setMeta(Meta meta) {
 		this.meta = meta;
 	}
+	
+	/**
+	 * @return void
+	 */
+	public void generateSerializerGav() {
```

Comment:
Pass this.getMeta().getSource() as a method parameter instead of calling multiple times.